1.0/tools/tests.md

```diff
@@ -0,0 +1,385 @@
+reverted at the end of each test.
+
+## AJAX
+
+<div class="yt-embed">
+  <google-youtube
+    videoid="_9qARcdCAn4"
+    autoplay="0"
+    rel="0"
+    fluid>
+  </google-youtube>
+</div>
+
+`wct` includes [Sinon](http://sinonjs.org/), which enables you to mock XHR
+requests and create fake servers. 
+
+Below is an example of a simple XHR unit test suite for 
+[`<iron-ajax>`](https://elements.polymer-project.org/elements/iron-ajax).
+Check out Sinon's documentation for more in-depth examples. 
+
+{% highlight html %}
+<!-- create test fixture template -->
+<test-fixture id="simple-get">
+  <template>
+    <iron-ajax url="/responds_to_get_with_json"></iron-ajax>
+  </template>
+</test-fixture>
+<script>
+  suite('<iron-ajax>', function() {
+    var ajax;
+    var request;
+    var server;
+    var responseHeaders = {
+      json: { 'Content-Type': 'application/json' }
+    };
+    setup(function() {
+      server = sinon.fakeServer.create();
+      server.respondWith(
+        'GET',
+        /\/responds_to_get_with_json.*/, [
+          200,
+          responseHeaders.json,
+          '{"success":true}'
+        ]
+      );
+    });
+    teardown(function() {
+      server.restore();
+    });
+    suite('when making simple GET requests for JSON', function() {
+      setup(function() {
+        // get fresh instance of iron-ajax before every test
+        ajax = fixture('simple-get');
+      });
+      test('has sane defaults that love you', function() {
+        request = ajax.generateRequest();
+        server.respond();
+        expect(request.response).to.be.ok;
+        expect(request.response).to.be.an('object');
+        expect(request.response.success).to.be.equal(true);
+      });
+      test('has the correct xhr method', function() {
+        request = ajax.generateRequest();
+        expect(request.xhr.method).to.be.equal('GET');
+      });
+    });
+  });
+</script>
+{% endhighlight %}
+
+The example above uses Chai's [`expect`](http://chaijs.com/api/bdd/) assertion
+style. 
+{: .alert .alert-info }
+
+## Run a subset of tests {#test-subsets}
+
+To run a subset of tests, create an HTML file and call `loadSuites()`. When
+running `wct`, specify the path to the HTML file as the first argument 
+(for example, `wct test/my-test-subset.html`.
+
+{% highlight html %}
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src=”../bower_components/webcomponentsjs/web-components-lite.js”></script>
+    <script src=”../bower_components/web-component-tester/browser.js”></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'basic.html',
+        'async.html'
+      ]);
+    </script>
+  </body>
+</html>
+{% endhighlight %}
+
+The argument to `loadSuites()` should be an array of strings. Each string
+should be a relative URL to a test suite. You can configure your tests 
+using query strings in the URLs. See [Test Shadow DOM](#shadow-dom)
+for an example.
+
+## Asynchronous tests
```

Comment:
I think this could be moved above "Preventing shared state with test fixture". Async tests are super common (and it uses assert, like the other tests in that section)